src/NuGet.Protocol.Catalog/Models/AlternatePackage.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class AlternatePackage
+    {
+        [JsonProperty("@id")]
+        public string Url { get; set; }
+
+        [JsonProperty("@type")]
+        public string Type { get; set; }
+
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        [JsonProperty("range")]
+        public string Range { get; set; }
```

Comment:
Is this a VersionRange?

Reply:
It is a version range, yet.

Reply:
Regarding renaming, I tried to make the models match the protocol as much as possible so that it would be easy to look at the protocol docs and know which property it is associated with in IntelliSense (which does not show `JsonProperty` values).